client/src/components/DataFiles/DataFilesProjectsList/DataFilesProjectsList.js

```diff
@@ -90,15 +90,20 @@ const DataFilesProjectsList = ({ modal }) => {
 
   if (error) {
     return (
-      <Message type="error">
-        There was a problem retrieving your Shared Workspaces
-      </Message>
+      <div styleName="root-placeholder">
+        <SectionMessage type="error">
+          There was a problem retrieving your Shared Workspaces.
+        </SectionMessage>
+      </div>
     );
   }
 
   return (
-    <div styleName="root">
+    /* !!!: Temporary bad indentation to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
+    <SectionTable styleName="root" contentShouldScroll manualContent>
       {!modal && <DataFilesProjectsSearchbar />}
+      <div className="o-flex-item-table-wrap">
       <InfiniteScrollTable
```

Comment:
Uh oh… I'll figure out what happenned. — __Task!__

Reply:
__Fixed__
I had to use accurate percentage widths—not non-percentage widths—for the columns of that table which is now `table-layout: fixed`.

- 26453642
     - [Projects 1000x700](https://user-images.githubusercontent.com/62723358/116720926-57564600-a9a2-11eb-844b-2be3b407585d.png)
    - [Projects 1920x700](https://user-images.githubusercontent.com/62723358/116720928-57eedc80-a9a2-11eb-94dc-7e12550eb98d.png)
    - [Projects 800x700](https://user-images.githubusercontent.com/62723358/116720930-58877300-a9a2-11eb-975d-4cde6a6b1dab.png)

Reply:
This change occurs because flex is no longer used for table layout, so the `table-layout: fixed` we had now kicks in. (See [`InfiniteScrollTable.scss`](https://github.com/TACC/Core-Portal/blob/task/FP-385-section-comp-welcome-msg--sections/client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss) for the relevant code.)